wp-content/plugins/zippicks-smart-search/includes/class-intent-classifier.php

```
@@ -0,0 +1,434 @@
+        // Normalize score
+        return $matches > 0 ? min($score / max($matches, 1), 1.0) : 0;
+    }
+    
+    /**
+     * Calculate utility score
+     * 
+     * @param string $query
+     * @param array $tokens
+     * @return float
+     */
+    private static function calculate_utility_score($query, $tokens) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$tokens'. (Unused Code Rules)

(UnusedFormalParameter)